src/main/groovy/com/smartbear/swagger/Swagger2TestCaseGenerator.groovy

```diff
@@ -0,0 +1,60 @@
+package com.smartbear.swagger
+
+import com.eviware.soapui.impl.rest.RestRequestInterface.HttpMethod
+import com.eviware.soapui.impl.rest.RestResource
+import com.eviware.soapui.impl.wsdl.WsdlProject
+import com.eviware.soapui.impl.wsdl.WsdlTestSuite
+import com.eviware.soapui.impl.wsdl.testcase.WsdlTestCase
+import com.eviware.soapui.impl.wsdl.teststeps.RestTestRequestStep
+import com.eviware.soapui.impl.wsdl.teststeps.registry.RestRequestStepFactory
+import com.eviware.soapui.model.testsuite.TestProperty
+import io.swagger.models.Operation
+import io.swagger.models.Path
+import io.swagger.models.parameters.AbstractSerializableParameter
+
+class Swagger2TestCaseGenerator {
+    static void generateTestCases(WsdlProject project, RestResource resource, Path path) {
+        WsdlTestSuite testSuite = createTestSuiteIfNotPresent(project)
+        WsdlTestCase testCase = testSuite.addNewTestCase("$resource.name-TestCase")
+        resource.restMethodList.each {
```

Comment:
well - for example I was thinking we create a "Valid HTTP Status Codes" assertion validating against the list of defined responses - and optionally we create a "Swagger Compliance Assertion" which we point at the specified Swagger definition (if the endpoint to the Swagger definition is a URL) - not sure what other assertions we could come up with based on metadata - but the extension I had started on would definitely get lost here.

This is fine for now - we can rework it if the need arises